.github/workflows/hemtt-build.yml

```diff
@@ -0,0 +1,27 @@
+name: "HEMTT Build"
+on:
+- pull_request
+
+jobs:
+  hemtt-build:
+    runs-on: windows-latest
+    steps:
+    - uses: actions/checkout@master
+    - name: Setup HEMTT
+      uses: arma-actions/hemtt@v1
+    #- name: Steam - Download

```

Comment:
It would be nice to have the steam tools as it would allow us to do some useful things